scipy/stats/_multivariate.py

```diff
@@ -6978,3 +6980,305 @@ def entropy(self):
 
         """
         return self._dist._entropy(self.dim, self.kappa)
+
+
+class normal_inverse_gamma_gen(multi_rv_generic):
+    r"""Normal inverse gamma distribution.
+
+    The normal inverse gamma distribution is the conjugate prior of a normal
+    distribution with unknown mean and variance.
+
+    Parameters
+    ----------
+    mu, lmbda, a, b  : array_like
+        Shape parameters of the distribution. See notes.
```

Comment:
I opted against using the string replacements common in this file. We can consider using them in some places.